zerotier/hooks/post-run

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+{{#if cfg.network ~}}
+  echo "Joining network {{cfg.network}}"
+
+  # wait for service to come up. something like habitat-sh/habitat#3721 would make this unnecessary
+  while [ ! -f /hab/svc/zerotier-one/data/zerotier-one.pid ] || [ ! -f /hab/svc/zerotier-one/data/zerotier-one.port ]; do
+    echo "Waiting for zerotier-one.pid and zerotier-one.port"
+    sleep 1
+  done
+
+  zerotier-cli join {{cfg.network}} || echo "Failed to join network"
+{{else}}
```

Comment:
Is network not a required configuration item? If not, perhaps the hook can just skip the `else` part of this.

Reply:
@predominant the client service is stateful and you can be a member of any number of networks at a time through join/leave commands. This configuration option might be better named `autojoin_network`. You can start the service without this configured and then use the CLI to join/create a network later.

I included the else because I found it valuable for telling the difference between a `network` config not making it down to the service and the client API never coming online